sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/_io/base.py

```diff
@@ -69,11 +69,21 @@ def __init__(self, meta: Union[Input, Output], data, **kwargs):
         :type data: Union[None, int, bool, float, str
                           azure.ai.ml.Input,
                           azure.ai.ml.Output]
+        :param original_data: default value of input/output, None means un-configured data.
```

Comment:
default_data?

Reply:
Actually it's not default data, for pipeline input, default data is stored in `self._data` field.

Reply:
The docstring doesn't match change in line 63

Reply:
Good catch!

Reply:
Updated doc string